libvcx/src/api/vcx.rs

```diff
@@ -137,8 +145,16 @@ pub extern fn vcx_init_issuer_config(command_handle: CommandHandle, config: *con
 
     trace!("vcx_init_issuer_config >>> config: {}", config);
 
+    let issuer_config = match serde_json::from_str::<IssuerConfig>(&config) {
+        Ok(issuer_config) => issuer_config,
+        Err(err) => {
+            error!("vcx_init_issuer_config >>> invalid configuration; err: {:?}", err);
```

Comment:
I like it, but it is a new style of error logging (using semicolon instead of a comma)

Reply:
updated